src/gpuarray_buffer_cuda.c

```diff
@@ -1217,9 +1217,8 @@ static gpukernel *cuda_newkernel(void *c, unsigned int count,
           return NULL;
         }
         if (compile_cache == NULL)
-          compile_cache = cache_twoq(16, 16, 16, 8, src_eq, src_hash, src_free,
-                                     bin_free);
-
+          compile_cache = cache_twoq(64, 128, 64, 8, src_eq, src_hash,
```

Comment:
It would be good to document that method in cache.h. It too me some time to understand it and I'm still not sure what the elasticity praameter is.